common/config/rush/pnpm-lock.yaml

```diff
@@ -9292,16 +9290,14 @@ packages:
       '@microsoft/api-extractor': 7.7.11
       '@opentelemetry/api': 0.10.2
       '@rollup/plugin-commonjs': 11.0.2_rollup@1.32.1
-      '@rollup/plugin-inject': 4.0.2_rollup@1.32.1
       '@rollup/plugin-json': 4.1.0_rollup@1.32.1
       '@rollup/plugin-multi-entry': 3.0.1_rollup@1.32.1
       '@rollup/plugin-node-resolve': 8.4.0_rollup@1.32.1
       '@rollup/plugin-replace': 2.3.4_rollup@1.32.1
       '@types/chai': 4.2.14
       '@types/mocha': 7.0.2
-      '@types/node': 8.10.66
+      '@types/node': 10.17.51
```

Comment:
/cc @xirzec

Reply:
I don't like this change.

@vxfield Can you try discarding this `pnpm-lock.yaml` file and copy the one in the master branch instead? 
And then run `rush update --recheck` and push the new changes?

Reference: https://github.com/Azure/azure-sdk-for-js/issues/11789

Reply:
done in PR #13612